src/highlight.js

```diff
@@ -25,7 +25,7 @@ import HTMLInjectionError from "./lib/html_injection_error.js";
 @typedef {import('highlight.js').HLJSPlugin} HLJSPlugin
 @typedef {import('highlight.js').PluginEvent} PluginEvent
 @typedef {import('highlight.js').HLJSOptions} HLJSOptions
-@typedef {import('highlight.js').LanguageFn} LanguageFn
+@typedef {import('highlight.js').LanguageFn} {import("highlight.js").LanguageFn}
```

Comment:
What makes LanguageFn different from all the other types listed here?

Reply:
LanguageFn is type of this function that takes HLJSApi type parameter and returns Language type output 

But in DocCommets it wasn't used correctly 
```javascript 
/** @type LanguageFn */
```
It doesn't assign the LanguageFn type to it

So, it should be changed to 
```javascript 
/** @type {import("highlight.js").LanguageFn}
```
It will import all types from highlight.js and then we select LanguageFn type from it and assign it to function to assign type in DocComments, type must be put between {} after @type

Reply:
> What makes LanguageFn different from all the other types listed here?

It was his type
I didn't do anything more, I just correctly type assignment

Reply:
The whole point of `@typedef {import('highlight.js').LanguageFn} LanguageFn` should be to allow us to use the short names later on in the file - otherwise what's the point in typedef?

Reply:
Understood

Reply:
> The whole point of `@typedef {import('highlight.js').LanguageFn} LanguageFn` should be to allow us to use the short names later on in the file - otherwise what's the point in typedef?

I have updated everything as you asked